packaging/network-manager-conf/microshift-nm.conf

```diff
@@ -1,6 +0,0 @@
-# This file configures NetworkManager to use the keyfile plugin
-# that will make OVS initialization work with IP static addressing
```

Comment:
This comment makes me think this is required for some configurations. Is that not actually the case?

Reply:
If you configure network manager to already have these plugins then there is no need to override anything.
This is the default config for nm:
```
# Configuration file for NetworkManager.
#
# See "man 5 NetworkManager.conf" for details.
#
# The directories /usr/lib/NetworkManager/conf.d/ and /run/NetworkManager/conf.d/
# can contain additional .conf snippets installed by packages. These files are
# read before NetworkManager.conf and have thus lowest priority.
# The directory /etc/NetworkManager/conf.d/ can contain additional .conf
# snippets. Those snippets are merged last and overwrite the settings from this main
# file.
#
# The files within one conf.d/ directory are read in asciibetical order.
#
# You can prevent loading a file /usr/lib/NetworkManager/conf.d/NAME.conf
# by having a file NAME.conf in either /run/NetworkManager/conf.d/ or /etc/NetworkManager/conf.d/.
# Likewise, snippets from /run can be prevented from loading by placing
# a file with the same name in /etc/NetworkManager/conf.d/.
#
# If two files define the same key, the one that is read afterwards will overwrite
# the previous one.
[main]
#plugins=keyfile,ifcfg-rh
```

Also note this line:
```
# If two files define the same key, the one that is read afterwards will overwrite
# the previous one.
```
So it might even override what it shouldnt. This config file was introduced in july 2022, so the content might be dated?

Reply:
I think this file was added to support the microshift-ovs-init.service. That service used to require NM to use keyfile plugin.  But now as we decoupled the br-ex bridge from the physical interface. I think we no longer need this file.